sceneManager.js

```diff
@@ -10,8 +10,19 @@ class SceneManager {
         this.gameOver = false;
 
         this.title = true;
+        this.credits = false;
         this.level = null;
 
+        this.menuSelect = {
```

Comment:
_I'm only putting this here to take note for the future_

While the menu works it might be good to simplify the logic for selection.

A way could be having an array of selection options and a selection index where you could do something akin to the "psuedo-code" below.

```javascript
const menuOptions = ["mario", "luigi", "credits"];
let selection = 0;

if (selectionDown) selection = (selection + 1) % menuOptions.length
if (selectionUp) selection = (selection - 1) % menuOptions.length
```

This way, you don't have to keep track of an Object and what flags are set and making sure they are updated in a valid manner.

However, it works right now and the selection menu won't be much more complex so this is a suggestion for a future edit.